src/Domain/Models/Configuration.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Models
+{
+    public class Configuration
```

Comment:
I would use a more specific name for this if possible.

Reply:
Nice catch, anything in mind? `ServiceConfiguration` / `HubConfigurationEntry` sound maybe better?

Reply:
I would go with `HubConfigurationEntry` or something like that.